salalib/salaprogram.cpp

```diff
@@ -1656,7 +1652,9 @@ SalaObj SalaCommand::connections(SalaObj graphobj, SalaObj param)
       }
    }
    else {
-      const Connector& connector = graphobj.data.graph.map.shape->getConnections()[graphobj.data.graph.node];
+      int idx = std::distance(graphobj.data.graph.map.shape->getAllShapes().begin(),
```

Comment:
I can't see the context here - do we need the `std::distance` here or are we in a case where keys are just indices?